roles/wildfly_validation/tasks/pidfile_validate.yml

```diff
@@ -8,6 +8,17 @@
   when:
     - wildfly_validation_systemd_pidfile is defined and wildfly_validation_systemd_pidfile | length > 0
 
+- name: "Check if {{ wildfly_instance_name }} service is running"
+  ansible.builtin.systemd:
+    name: "{{ wildfly_instance_name }}"
+  register: service_status
```

Comment:
checking the status of the service is already done by the populate service.

Reply:
thanks did the changes.